### PR TITLE
Create LIT7087QeneDaggazmacc.xml

### PR DESCRIPTION
**Pull request by karljonaskarlsson**
Yet another qene from Uppsala: https://www.alvin-portal.org/alvin/imageViewer.jsf?dsId=ATTACHMENT-0350&pid=alvin-record:280997

It refers to a "Daǧǧāzmāčč Wǝbe" – do you think the identification that I propose is possible or do you have any better suggestions?

### CODE REVIEW

File: new/LIT7087QeneDaggazmacc.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7087QeneDaggazmacc" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ደጃዝማች፡ ውቤ፡ ኢታብዝህ፡ ኀዘነ፡ ልቡ<choice><sic>ነ</sic><corr resp="JK">ና</corr></choice>፡ 
+                    በሞተ፡ አሀዱ፡ ወልድከ፡ እንበለ፡ ጊዜሁ፡ <sic resp="JK">ወመዋዕሁ</sic>፤</title>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT7087QeneDaggazmacc.xml:9]: It is ሉ / ወመዋዕሉ in the end of the verse.

File: new/LIT7087QeneDaggazmacc.xml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+                <title xml:lang="gez" corresp="#t1" type="normalized">Daǧǧāzmāčč Wǝbe ʾi-tābzǝh ḫazana lǝbbunā ba-mota ʾahadu waldǝka ʾǝnbala gizehu wa-mawāʿǝlu (Qǝne possibly of the mawaddǝs-type)</title>
+                <title xml:lang="en" corresp="#t1">Daǧǧāzmāčč Wǝbe, do not multiply the grief of understanding because of the death of your only son outside of its time and days…</title>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT7087QeneDaggazmacc.xml:11]: I propose:
Daǧǧāzmāčč Wǝbe, do not multiply the grief of watching over the death of your only son except for his time and his days...

File: new/LIT7087QeneDaggazmacc.xml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Qǝne</foreign> without type indication in the manuscript. It has eight lines, meaning
+                    that it can be classified as a <foreign xml:lang="gez">mawaddəs qəne</foreign> according to
+                    <bibl><ptr target="bm:Guidi1900Qene"/></bibl>. 
+                    In the first line, it refers to a "<foreign xml:lang="gez">Daǧǧāzmāčč</foreign> Wǝbe", possibly
+                    <persName ref="PRS10100WebeAt"/>.
```
Comment (CarstenHoffmannMarburg) [inline new/LIT7087QeneDaggazmacc.xml:42]: Yes, I think, this identification is possible and correct to mark it this way.

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: new/LIT7087QeneDaggazmacc.xml (hunk 1 above)
Comment (eu-genia) [reply new/LIT7087QeneDaggazmacc.xml:9]: remove mark up in `title`

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Review by nafisa-valieva — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**